code/game/gamemodes/cult/runes.dm

```diff
@@ -460,7 +487,7 @@ var/list/teleport_runes = list()
 
 //Ritual of Dimensional Rending: Calls forth the avatar of Nar-Sie upon the station.
 /obj/effect/rune/narsie
-	cultist_name = "Tear Reality (God)"
+	cultist_name = "Summon Your God"
```

Comment:
Personally I think you should keep it Tear Reality (God) as it fits the naming scheme better than Summon Your God.